src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -339,6 +340,13 @@ namespace HeatBalanceManager {
 		// Get the heat balance input at the beginning of the simulation only
 		if ( ManageHeatBalanceGetInputFlag ) {
 			GetHeatBalanceInput(); // Obtains heat balance related parameters from input file
+
+			// Surface octree setup
+			//  The surface octree holds live references to surfaces so it must be updated
+			//   if in the future surfaces are altered after this point
+			surfaceOctree.init( DataSurfaces::Surface ); // Set up surface octree
```

Comment:
@nealkruis That's a good point -- I'll add that. I wanted to even skip initializing it if daylighting wasn't active but the logic to detect that seemed complex and @vidanovic suggested to not bother. The octree setup doesn't take a signficant amount of time but it is still sloppy to do it if we'll never use it.